mxnet/training/docker/1.6.0/py3/Dockerfile.gpu

```diff
@@ -159,7 +159,7 @@ RUN ${PIP} install --no-cache --upgrade \
     sagemaker-experiments==0.1.7 \
     PyYAML==5.3.1 \
     mpi4py==3.0.2 \
-    "sagemaker-mxnet-training<4" \
+    sagemaker-mxnet-training==4.2.0 \
```

Comment:
Could you set this to "sagemaker-mxnet-training<5"?

We need to be able to update the version of this package without having to manually change this version frequently.

Reply:
Addressed